src/apps/cores/exceptions.py

```diff
@@ -74,3 +77,9 @@ def __init__(self, message='This method is not allowed.'):
 class InternalServerError(BaseError):
     def __init__(self, message='Internal server error.'):
         super().__init__(code=500, message=message)
+
+
+class UnprocessableEntity(BaseError):
+    def __init__(self, error=None):
+        self.message = error.description
+        super().__init__(code=400, message=self.message)
```

Comment:
super trước r assign sau cho nó đỡ phá cách ông êi :v: 
```suggestion
        super().__init__(code=400, message=self.message)
         self.message = error.description
```